rust/kernel/time.rs

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Time representation in the kernel.
+//!
+//! C headers: [`include/linux/time64.h`](../../include/linux/time64.h)
+
+use crate::{bindings, error::Result};
+
+/// A [`Time`] instance at the Unix epoch.
+pub const UNIX_EPOCH: Time = Time {
```

Comment:
I believe people will give you a hard time if they see this is called `Time` ;-) What about `TimeSpec64`? Similar to C side name, and open to any better naming ;-)